casepro/msg_board/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django_comments.models import Comment
+
+
+class MessageBoardComment(Comment):
```

Comment:
It determines the database table used. Since we're creating our own comment subclass, it's better it goes in our own database table in our own app.